app/controllers/users_controller.rb

```diff
@@ -0,0 +1,12 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+    @upcoming_events = current_user.attended_events.upcoming
+    @past_events = current_user.attended_events.previous
+  end
+
+  def show
+    @user = User.find(params[:id])
+    @user = current_user
+  end
```

Comment:
Move the `@upcoming_events` and `@past_events` instance variables from the index to show action since it is the user's show page that displays events the user has created and events that the user is attending to separated in past and upcoming